tests/bootstrap.php

```diff
@@ -6,6 +6,7 @@
 require_once __DIR__ . '/../vendor/autoload.php';
 require_once __DIR__ . '/PHPStan/Rules/AlwaysFailRule.php';
 require_once __DIR__ . '/PHPStan/Rules/DummyRule.php';
+require_once __DIR__ . '/PHPStan/Type/PhpDecimalOperatorTypeSpecifyingExtension.php';
```

Comment:
This should not be necessary - already loaded thanks to `autoload-dev` in composer.json